spec/system/candidate_interface/signup_and_signin/candidate_account_spec.rb

```diff
@@ -14,6 +14,7 @@
     and_i_submit_my_email_address
     then_i_receive_an_email_with_a_signup_link
     when_i_click_on_the_link_in_my_email
+    when_i_confirm_account_creation
```

Comment:
Don't mind either way, but I think `and_i_confirm_account_creation` and 'and_i_confirm_sign_in' would be more readable here and more in keeping with the rest of the app.

Reply:
Done, thanks!